sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/RollingUpgradePolicy.cs

```diff
@@ -138,29 +138,38 @@ public RollingUpgradePolicy(int? maxBatchInstancePercent = default(int?), int? m
         /// </exception>
         public virtual void Validate()
         {
```

Comment:
not sure why this file is being updated. does your swagger change have anything related to this? 
if not, could you please take this change out of the PR